extension/chrome/elements/compose-modules/compose-draft-module.ts

```diff
@@ -150,6 +180,38 @@ export class ComposeDraftModule extends ViewModule<ComposeView> {
     }
   }
 
+  private isLocalDraftId = (draftId: string) => {
+    return !!draftId.match(this.localDraftPrefix);
+  }
+
+  private localDraftCreate = async (mimeMsg: string, threadId: string, recipients: string[], subject: string) => {
```

Comment:
There may be an error in this method. A few lines below there is an `if (threadId)` and `else`. But regardless if there is thread id or not, this `threadId` is used as part of `draftId`. It appears that all new messages have the same draft id, because there is no thread to differentiate them. Is this intentional? If yes, it would be good to add a comment with the reasoning / intended behavior

Reply:
Yup, that's intentional because the new message (not reply) will always have the same local draft ID, which is now `this.draftModule.localNewMessageDraftId`.